lib/utils/properties-panel.js

```diff
@@ -289,6 +298,10 @@ export function getErrorMessage(id, report) {
     return 'Cannot be an expression.';
   }
 
+  if (data.type === ERROR_TYPES.SECRET_EXPRESSION_INVALID) {
+    return 'Must be a valid secret.';
```

Comment:
I don't think `Must` is the correct word here. It could also be any other value that is not a secret. I would use the same wording as in the errors panel and phrase it as a hint

![image](https://github.com/camunda/linting/assets/21984219/f4526ab8-e1fe-4a2f-b08c-0205008e8bbd)

```suggestion
    return 'Is not a valid secret.';
```